### DISCUSSION TITLE
Mount drive at boot

### DISCUSSION THREAD

**Discussion by safe-canvas** [itsToggle/plex_debrid#633] [Q&A]
I am trying to automate this using task scheduler on my Synology NAS, to mount after reboot. It works if I manually run the task but it will not work automatically after reboot. Does anyone know why or what I need to add extra to this command to run automatically? I am running Plex in Docker and the container will not start if this drive is not mounted.

docker volume create realdebrid -d rclone -o type=realdebrid -o realdebrid-api_key=MyAPIKeyHere -o allow-other=true -o dir-cache-time=10s

Thank you

**Comment by demondamz** [comment]
Create a systemctl service that will mount the volume for you
